### PR TITLE
Doc/issue 370

### PR DESCRIPTION
**Pull request by MaxJPRey**
Rename some variables to respect the PEP8.
This work was done while reviewing the issue #370 .

### CODE REVIEW

**Review by akaszynski — APPROVED** [review]
LGTM.

File: examples/01-advanced-postprocessing/Maxwell2D_Transient.py [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-This tutorial shows how you can use PyAedt to create a project in Maxwell2D
-and run a transient simulation.
-This example needs PyVista, numpy and matplotlib, to be installed on the
-machine to provide advanced post processing features.
-This example runs on Windows only using CPython.
+-------------------
+This example shows how you can use PyAedt to create a project in
+in Maxwell 2D and run a transient simulation.
 
 To provide the advanced postprocessing features needed for this example, Matplotlib, NumPy, and
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Maxwell2D_Transient.py]: You could consider providing links to these libraries (pypi or docs)

Comment (MaxJPRey) [reply examples/01-advanced-postprocessing/Maxwell2D_Transient.py]: @akaszynski Let me know if my latest modifications look good to you.

File: examples/01-advanced-postprocessing/Maxwell2D_Transient.py [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
-in Maxwell 2D and run a transient simulation.
+This example shows how you can use PyAedt to create a project in Maxwell 2D
+and run a transient simulation.
 
-To provide the advanced postprocessing features needed for this example, Matplotlib, NumPy, and 
-PyVista must be installed on the machine.
+To provide the advanced postprocessing features needed for this example,
+Matplotlib: <https://pypi.org/project/matplotlib/>,
+NumPy: <https://pypi.org/project/numpy/>,
+and PyVista https://pypi.org/project/pyvista/ must be installed on the machine.
```
Comment (akaszynski) [inline examples/01-advanced-postprocessing/Maxwell2D_Transient.py:11]: Please use rst style documentation for the links:

```suggestion
The following libraries are required for the advanced post-processing features
used in this example:

- `matplotlib <https://pypi.org/project/matplotlib/>`_
- `NumPy <https://pypi.org/project/numpy/>`_
- `PyVista https://pypi.org/project/pyvista/`_

Install these with:

.. code::

   pip install numpy pyvista matplotlib

```

File: examples/01-advanced-postprocessing/Maxwell2D_Transient.py (hunk 1 above)
Comment (akaszynski) [reply examples/01-advanced-postprocessing/Maxwell2D_Transient.py]: Please see my comments.

**Review by PipKat — APPROVED** [review]

File: examples/01-advanced-postprocessing/Maxwell2D_Transient.py (hunk 2 above)
Comment (MaxJPRey) [reply examples/01-advanced-postprocessing/Maxwell2D_Transient.py]: @akaszynski I did that the first time and then switched back to the current version. Thanks for the feedback. I will do the modifications.

**Pull request merged**